K8s-deployment/Charts/resource-server/values.yaml

```diff
@@ -342,2097 +342,6 @@ apiServer:
-  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
-  ##
-  podAffinityPreset: ""
-  ## @param async.podAntiAffinityPreset Pod anti-affinity preset. Ignored if `async.affinity` is set. Allowed values: `soft` or `hard`
-  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
-  ##
-  podAntiAffinityPreset: ""
-  ## Node async.affinity preset
-  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
-  ##
-  nodeAffinityPreset:
-    ## @param async.nodeAffinityPreset.type Node affinity preset type. Ignored if `async.affinity` is set. Allowed values: `soft` or `hard`
-    ##
-    type: ""
-    ## @param async.nodeAffinityPreset.key Node label key to match. Ignored if `async.affinity` is set
-    ##
-    key: ""
-    ## @param async.nodeAffinityPreset.values Node label values to match. Ignored if `async.affinity` is set
-    ## E.g.
-    ## values:
-    ##   - e2e-az1
-    ##   - e2e-az2
-    ##
-    values: []
-  ## @param async.affinity Affinity for async pods assignment
-  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-  ## NOTE: `async.podAffinityPreset`, `async.podAntiAffinityPreset`, and `async.nodeAffinityPreset` will be ignored when it's set
-  ##
-  affinity: {}
-  ## @param async.nodeSelector Node labels for async pods assignment
-  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
-  ##
-  nodeSelector:
-
-  ## @param async.tolerations Tolerations for async pods assignment
-  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-  ##
-  tolerations: []
-  ## @param async.updateStrategy.type async statefulset strategy type
-  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
-  ##
-  updateStrategy:
-    ## StrategyType
-    ## Can be set to RollingUpdate or OnDelete
-    ##
-    ##type: RollingUpdate
-  ## @param async.priorityClassName async pods' priorityClassName
-  ##
-  priorityClassName: ""
-  ## @param async.schedulerName Name of the k8s scheduler (other than default) for async pods
-  ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
-  ##
-  schedulerName: ""
-  ## @param async.lifecycleHooks for the async container(s) to automate configuration before or after startup
-  ##
-  lifecycleHooks: {}
-  ## @param async.extraEnvVars Array with extra environment variables to add to async nodes
-  ## e.g:
-  ## extraEnvVars:
-  ##   - name: FOO
-  ##     value: "bar"
-  ##
-  extraEnvVars: []
-  ## @param async.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for async nodes
-  ##
-  extraEnvVarsCM: "rs-env"
-  ## @param async.extraEnvVarsSecret Name of existing Secret containing extra env vars for async nodes
-  ##
-  extraEnvVarsSecret: "rs-s3-env"
```

Comment:
**We still need to pass this for the async verticle to work, please add it in above resourceServer block.**